scripts/psiminer_config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "format": "code2seq",
+  "problem": "method name prediction",
+
+  "resolveTypes": true,
+  "compressOperators": false,
+  "removeKeyword": false,
+  "removeComments": false,
+  "removeJavaDoc": true,
+  "compressTree": true,
+
+  "maxPathWidth": 4,
+  "maxPathLength": 9,
+  "maxPathsInTrain": 1000,
+  "maxPathsInTest": 200,
+  "nodesToNumbers": true,
+
+  "filters": ["constructor"],
+
+  "printTrees": false
+}
```

Comment:
Please add NL at the end of the file (e.g., `cat path/to/file` will be prettier)